app/views/plugins/ecommerce/partials/checkout/_user_info.html.erb

```diff
@@ -2,6 +2,7 @@
     cart ||= @cart
     billing_address = cart.get_meta("billing_address", {})
     shipping_address = cart.get_meta("shipping_address", {})
+    shipping_address = {} unless shipping_address
```

Comment:
What about changing the line `shipping_address = cart.get_meta("shipping_address", {})` to `shipping_address = cart.get_meta("shipping_address", {}) || {}`